### DISCUSSION TITLE
🧩 FEATURE REQUEST: Add filters to Search Feed

### DISCUSSION THREAD

**Discussion by sansk** [dailydotdev/daily#469] [Feature Requests]
### Description
Add Filters to Search feed.

### Solution
When searching, the search feed displays articles without any filter. 
![image](https://user-images.githubusercontent.com/3327904/153844718-06ede9e9-1100-4fa9-8059-786a5f35abb0.png)

Would love to have filters based on 'Date', 'Most Popular' or 'Most Upvoted', 'Most Commented' etc.
Something like in the home feed
![image](https://user-images.githubusercontent.com/3327904/153845482-53a9aebc-f2a0-4dd5-801f-d3f4cd981ea2.png)

**Comment by rebelchris** [comment]
Hi @sansk great suggestions
I'll discuss it with the team and will convert this into a discussion for now.

**Comment by idoshamun** [comment]
Let's keep the convo here: https://github.com/dailydotdev/daily/discussions/294
